build.gradle

```diff
@@ -4,8 +4,8 @@ plugins {
 
     // For JShell support
     // https://gitlab.com/barfuin/gradle-taskinfo
-    id 'org.barfuin.gradle.taskinfo' version '1.1.1'
-    id "com.github.mrsarm.jshell.plugin" version "1.1.0"
+    id 'org.barfuin.gradle.taskinfo' version '2.1.0'
```

Comment:
Upgrading Gradle required upgrading gradle-taskinfo. I decided to also update jshell.plugin at the same time, though it was not technically required.

Reply:
To be honest though, I have no idea how to tell if JShell still works with these changes.

Reply:
I wonder how/why JShell was used here? I don't see that 'jshell' task is used anywhere. Maybe we don't need it at all...

Reply:
Sorry, I didn't notice the comment before I hit merge.

@ddietterich add the JShell support. Dan, is it something that you use?